repos/rust-lang/rust-enhanced.toml

```diff
@@ -0,0 +1,7 @@
+org = "rust-lang"
+name = "rust-enhanced"
+description = "The official Sublime Text 3 package for the Rust Programming Language"
```

Comment:
This has been outdated for a while, but I procrastinated since asking an admin to fix it was just too much friction.

```suggestion
description = "The official Sublime Text 4 package for the Rust Programming Language"
```

Reply:
Changed.